assets/css/tasks/_blog-tasks.css

```diff
@@ -0,0 +1,71 @@
+/* POSTS PAGE */
+li:first-child { /* The :first-child CSS pseudo-class represents the first element among a group of sibling elements. */
+    /* margin-top; reset the value for this margin */
+    margin-top: 0%;
+}
+
+li:last-child {
+    /* margin-bottom; reset the value for this margin */
+    margin-bottom: 0%;
+}
+
+.post-box__img {
+    /* the element should occupy the half of the available width. Take into account the width of a content and a margin */
+    /* width; */
+    /* float; set element on left */
+    /* display; the element should be type of element which occupy the entire available place */
+    /* margin-right; */
+    width: 45%;
+    float:left;
+    margin-right: 10%;
+    display:block;
+}
+
+.post-box__img img {
+    /* width; the element should occupy the entire available width */
+    width: 100%;
+
+}
+
+.post-box__description {
+    /* float; set element on left */
+    /* width; the element should occupy the half of the available width */
+ 
+    width: 45%;
+    float:left;
+}
+
+.read-more-button {
+    /* color; */
+    color: purple;
+    display:inline; 
```

Comment:
Add space between property's name and its value.
May be there is some way to apply general formating css rules. But I've never use any. We can look for it. And it will be good practice to remember about proper code formating :)